distribution/docs/src/main/jdocs/content/_plugins/pre-query-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,16 @@
+:type: pluginIntro
+:status: published
+:title: Pre-Federated-Query Plugins
```

Comment:
❓ Should we name the file `pre-federated-query-plugin-intro-contents.adoc` to match the content?